src/client/Microsoft.Rest.ClientRuntime/ServiceClient.cs

```diff
@@ -98,12 +124,12 @@ private string FrameworkVersion
             {
                 if (string.IsNullOrEmpty(_fxVersion))
                 {
-                    Assembly assembly = typeof(Object).GetTypeInfo().Assembly;
+                    Assembly assembly = typeof(Object).GetTypeInfo().Assembly;                    
```

Comment:
Looks like trailing spaces where the only difference, if you're going to make another change you might as well remove them so there isn't any change